catkin_ws/src/sensors/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(sensors)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  rosserial_arduino
+  rosserial_client
+  std_msgs  
+)
+
+catkin_package( CATKIN_DEPENDS
+  roscpp
+  rosserial_arduino
```

Comment:
I think you're missing rospy in these dependencies (the imu has rospy in this part of the CMakeLists

Reply:
same for the find_package() part as well